src/FondOfSpryker/Zed/CompanyUserCompanyAssigner/Business/Model/CompanyRoleInterface.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+namespace FondOfSpryker\Zed\CompanyUserCompanyAssigner\Business\Model;
+
+use Generated\Shared\Transfer\CompanyUserTransfer;
+
+interface CompanyRoleInterface
+{
+    /**
+     * @param \Generated\Shared\Transfer\CompanyUserTransfer $companyUserTransfer
+     *
+     * @return void
+     */
+    public function updateNonManufacturerUsersCompanyRole(
```

Comment:
```suggestion
    public function updateCompanyRolesOfNonManufacturerUsers(
```

Reply:
@daniel-rose i've made the desired change